src/OrderInventory.ts

```diff
@@ -0,0 +1,103 @@
+import { genericCleanup } from "./util";
+
+/**
+ * Sort Inventories by Code
+ * TODO: only grab the inventory where the button was pressed
+ */
+export class OrderInventory {
+  private sortByCode: boolean = false;
+  private tag = "sort-by-code";
+  private cleanups: Array<() => void> = [];
+
+  cleanup() {
+    this.cleanups.forEach((f, i) => {
+      f();
+      delete this.cleanups[i];
+    });
+    genericCleanup(this.tag);
+  }
+
+  run() {
+    this.sortInventoriesByCode();
+    this.addSortByCodeButton();
+  }
+
+
+
+  private sortInventoriesByCode() {
+    if (this.sortByCode) {
+      const inventories = Array.from(
+        document.querySelectorAll("div[class^='InventoryView__grid__'")
+      );
+
+      inventories.forEach(inventory => {
+        let sorted = Array.from(inventory.children).sort(function(
+          a: HTMLElement,
```

Comment:
This is actually wrong, cause `inventory.children` might not only include HTMLElement but returns Element instead. 
It's perfectly fine in our use case, cause we know, that this is correct, but I guess otherwise a filter would be required :)